packages/jest-circus/src/__tests__/circusItFailingTestError.test.ts

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import type {Global} from '@jest/types';
+
+let circusIt: Global.It;
+let circusTest: Global.It;
+
+// using jest-jasmine2's 'it' to test jest-circus's 'it'. Had to differentiate
+// the two with this alias.
+
```

Comment:
If the feature does not work on `jest-jasmine2`, probably it is not involved here. I think this comment is unnecessary. All clear what is happening.

```suggestion
```